docs/create-custom-page.md

```diff
@@ -0,0 +1,95 @@
+---
+id: create-custom-page
+title: Create a custom page
+---
+
+When creating an online store, you often have to create whole new pages specific to your store.
+
+Let's look at how it works by adding a "Hello World" page to Front Commerce Lite. This page will be accessible at `/greetings` and will display `Hello World`.
+
+## TL;DR
+
+In order to do so, we need to:
+
+1.  Create our component page in our theme
+2.  Define the associated route in our router
+
+## 1. Create a new page component
+
+First, let's create a new component in our theme. This component will be the one telling our application what to display on the `/greetings` page.
+
+In order to keep our code clean, we will put this component into the `pages` folder of our theme: `src/web/theme/pages/<our-page>.js`.
+
+> **Front Commerce:** If we were to use the paid version, the page should be added at the same path but within our custom module rather than in core modules.
+
+Once created, this component should render a basic "Hello World" message.
+
+```js
+// src/web/theme/pages/Greetings.js
+import React, { Component } from "react";
+import { H1 } from "theme/components/atoms/Typography/Heading";
+
+class Greetings extends Component {
+  render() {
+    return <H1>Hello world</H1>;
+  }
+}
+
+export default Greetings;
+```
+
+## 2. Define the associated route
+
+Once the component is created, we must declare our new route. A route is a React element that specifies what to display if a specific path is matched. In our case, we want it to render the `Greetings` component when the user is on `/greetings` URL. It uses [React Router](https://reacttraining.com/react-router/web/guides/philosophy), the most popular routing library in the React ecosystem.
+
+It looks like this:
```

Comment:
I am not sure this part is really needed. It may mislead readers who will try to use it « as is » and copy/paste the code without reading.

Maybe using a diff syntax in the `Routes.js` file would be more straightforward.